### PR TITLE
Fix some assertions on GLES2

### PR DESCRIPTION
**Pull request by loganmc10**
GLSL ES 1.00 doesn't like when you initialize a uniform, and uDepthScale is never used in GLES2, and it was giving me an assertion error

### CODE REVIEW

**Comment by loganmc10** [comment]
This fixes https://github.com/gonetz/GLideN64/issues/1010

File: src/OpenGL.cpp [hunk 1]
```diff
@@ -399,8 +399,10 @@ void OGLRender::TexrectDrawer::init()
 	assert(m_textureBoundsLoc >= 0);
 	m_enableAlphaTestLoc = glGetUniformLocation(m_programTex, "uEnableAlphaTest");
 	assert(m_enableAlphaTestLoc >= 0);
+#ifndef GLES2
 	m_depthScaleLoc = glGetUniformLocation(m_programTex, "uDepthScale");
 	assert(m_depthScaleLoc >= 0);
+#endif
 	glUseProgram(0);
```
Comment (gonetz) [inline src/OpenGL.cpp]: Please move these changes to another commit. Different issues should be fixed by separate commits. Both commits can be in one pull request.

**Comment by loganmc10** [comment]
Ok I've split it into 2 commits

**Comment by gonetz** [comment]
Thanks. Merged.
